app/jsx/components/PaginationComp.jsx

```diff
@@ -96,8 +98,10 @@ class PaginationComp extends React.Component {
       return (
         <div className="c-pagination">
           <a href="" className="c-pagination__prevnext" onClick={this.previous}>Previous</a>
-          <a href="" className="c-pagination__item" onClick={this.first}>1</a>
-          <span className="c-pagination__ellipses">&hellip;</span>
+          { (pages > 3) && 
+            [<a key="0" href="" className="c-pagination__item" onClick={this.first}>1</a>,
+             <span key="1" className="c-pagination__ellipses">&hellip;</span>]
+          }
```

Comment:
If you need an example of search results that give you less than 4 pages worth: "crepuscular sheep"